docker-compose.yml

```diff
@@ -12,6 +12,12 @@ services:
       - "8080:8080"
     environment:
       - PYTHONUNBUFFERED=1
+      - DB_ENGINE=mysql.connector.django
+      - DB_NAME=app_db
+      - DB_USER=app_user
+      - DB_PASSWORD=1234
```

Comment:
Storing sensitive information such as database passwords directly in the `docker-compose.yml` file is not recommended due to security concerns. Consider using Docker secrets or environment files to manage sensitive data securely.